src/main/kotlin/sliv/tool/catalogue/controller/CatalogueController.kt

```diff
@@ -0,0 +1,26 @@
+package sliv.tool.catalogue.controller
+
+import sliv.tool.catalogue.extractLayers
+import sliv.tool.catalogue.model.CatalogueModel
+import sliv.tool.project.model.LayerKind
+import sliv.tool.project.model.ProjectFrame
+import sliv.tool.project.model.ProjectLayer
+import sliv.tool.scene.SceneFacade
+import sliv.tool.scene.controller.SceneController
+import tornadofx.Controller
+
+class CatalogueController: Controller() {
+    val model = CatalogueModel()
+
+    private val sceneController: SceneController by inject()
+    private val sceneFacade = SceneFacade(sceneController)
```

Comment:
why facade is created here? maybe it have to be injected instead of controller?

Reply:
We can't use injection for classes which are not View or Controller. Maybe we can use singletone?

Reply:
we can use custom di)) lets create singleton for now -- read about ServiceLocator pattern